Casks/lilypond-unofficial.rb

```diff
@@ -0,0 +1,45 @@
+cask 'lilypond-unofficial' do
+  version '2.20.0'
+  sha256 '655bb00970279ae72d3c7dfe19186db9025f0cb5eae7e3c4485c1b3b1116bc69'
+
+  # bintray.com/marnen/lilypond-darwin-64/download_file?file_path=lilypond- was verified as official when first introduced to the cask
+  url "https://bintray.com/marnen/lilypond-darwin-64/download_file?file_path=lilypond-#{version}.build20200311175017-darwin-64.tar.gz"
+  # appcast 'https://lilypond.org/macos-x.html'
```

Comment:
Would `appcast 'https://bintray.com/marnen/lilypond-darwin-64/lilypond-2.20.0#release'` be suitable? It may also help to get the maintainer @marnen involved.

Reply:
I don't know, on this page, http://lilypond.org/macos-x.html the build of @marnen is officially offered

Reply:
> officially offered

I don't know about that, because the language on that page you linked should seem very clear to the average Joe. 

> **Unofficial** 64-bit application bundles for macOS 10.15 are available at https://bintray.com/marnen/lilypond-darwin-64. (emphasis added)

But that's besides my point. Please see the fifth bullet point under [the different ways the `appcast` can be determined.](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

> - An appcast can be _any URL_ hosted by the app’s developer _that changes every time a new release is out **or** that contains the version number of the current release_ (e.g. a download HTML page). Webpages that only change on new version releases are preferred, as are sites that do not contain previous version strings (i.e. avoid changelog pages if the download page contains the current version number but not older ones). (emphasis added)

Reply:
Ok, i change `appcast`, thanks pour this point of vue.